src/sentry/sentry_metrics/consumers/indexer/batch.py

```diff
@@ -175,7 +175,7 @@ def filter_messages(self, keys_to_remove: Sequence[PartitionIdxOffset]) -> None:
             "sentry_metrics.indexer.process_messages.dropped_message",
             amount=len(keys_to_remove),
             tags={
-                "reason": "cardinality_limit",
+                "reason": "cardinality_limit_legacy",
```

Comment:
you can just remove this metric IMO. the codepath you're adding the metric to gets executed for every usecase anyway, right?

Reply:
Yes, but I would like to verify that the old and new numbers matches up before removing it.